ral-facilities/scigateway
# The Images on the Help page need aria labels #815

## Description
Automated Accessibility checker found that the Images on the contact page don't have aria labels. 

![Screenshot 2021-11-12 131218](https://user-images.githubusercontent.com/83226114/141472648-3499eb22-1c08-4c1c-930c-32340749466a.png)
**Acceptance criteria**:

## Testing instructions
- Check for human readability 
- Check using a screen reader 


## Agile board tracking
connect to #{issue number}
